<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>别在这立法典——发病作文一键生成2.0 by ikcye</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>首先选择一部法典的模板</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在这里查阅你的专属法典吧！</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在这里替换你法典的内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个预设的替换关键词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>或者...自定义替换关键词</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCustomKeyword">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>你将要把整个模板内的</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKeyword">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>某个关键词</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>320</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>替换为:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineReplacement">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCopy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>复制法典到剪切板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReplace">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>就在这立法典！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭法典生成器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRefreshList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新模板列表</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listKeyword">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>256</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEggs">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>已发现彩蛋: 0</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.png</normaloff>icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMenu">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
